src/views/webview.rs

```diff
@@ -447,6 +500,124 @@ impl WebView {
         self.view.set_zoom_level(level - 0.1);
         (self.view.get_zoom_level() * 100.0) as i32
     }
+
+
+    /// Set the cookie accept policy.
+    fn set_cookie_accept(&self, cookie_accept: &CookieAcceptPolicy) {
+        let cookie_manager = self.view.get_context()
+            .and_then(|context| context.get_cookie_manager());
+        if let Some(cookie_manager) = cookie_manager {
+            cookie_manager.set_accept_policy(cookie_accept.to_webkit());
+        }
+    }
+
+    /// Adjust the webkit settings.
+    pub fn setting_changed(&self, setting: AppSettingsVariant) {
```

Comment:
I'm not a fan either, I just moved it because it made more sense to keep the views contained in their own file and when they were large enough turn it into something like `app/mod.rs`. I can move the settings out to something like `webview_settings` though